apps/src/lib/client/tx.rs

```diff
@@ -721,19 +739,22 @@ pub async fn submit_init_validator<'a>(
         description,
         website,
         discord_handle,
-        validator_vp_code_hash: extra_section_hash,
     };
 
     // Put together all the PKs that we have to sign with to verify ownership
-    let mut all_pks = data.account_keys.clone();
+    let account = namada_sdk::rpc::get_account_info(namada.client(), &address)
+        .await?
+        .expect("TODO");
```

Comment:
fill in this TODO

Reply:
done in [9122528](https://github.com/anoma/namada/pull/2208/commits/9122528e0aab864fa440b0262e049c473ca5a292)